README.md

```diff
@@ -56,6 +56,8 @@ Note that the external-resizer does not scale with more replicas. Only one exter
 
```

Comment:
Can you also update the image location to k8s.gcr.io/sig-storage/csi-resizer

Reply:
done.